### ISSUE TITLE
about loss in channel pruning gpu

### ISSUE THREAD

**Issue by chengyuz** [Tencent/PocketFlow#226]
what the loss would be like in channel pruning gpu:
INFO:tensorflow:iter 893: nnz-chns = 3 | loss = 1.50e+11 | lr = 1.53e-15 | percentile = 89.30
the loss is too large in compress self-defined models.

**Comment by jiaxiang-wu** [comment]
The scale of loss depends on the scale of feature maps. Can you measure feature map's L2-norm for comparison?

**Comment by jiaxiang-wu** [comment]
Any updates?
